release/release.sh

```diff
@@ -127,7 +127,8 @@ function get_release_notes() {
     $found && notes+=("$line")
   done < CHANGELOG.md
   # The sed here removes leading blank lines
-  local clean="$(printf "%s\n" "${notes[@]}" | sed '/./,$!d')"
+  local clean
+  clean="$(printf "%s\n" "${notes[@]}" | sed '/./,$!d')"
```

Comment:
I've been meaning to go in and fix this one.. Thanks for getting it too.

Reply:
no problem.